### DISCUSSION TITLE
Geometry messed up on launch

### DISCUSSION THREAD

**Discussion by drdozer** [zed-industries/zed#21978] [Feature Requests]
Hey - I didn't want to open an issue until I knew if this was just me. I'm on x11, devuan. When zed launches, it starts up with weird window geometry and status. I think it is thinking it is maximised. But it isn't. It also isn't positioned where it was when I closed the last session. I have a 2 screen setup. Is this just me? Is there a geometry persistence file I can delete to try to fix it?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
